about.css

```diff
@@ -0,0 +1,661 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,300;1,400;1,700&family=Poppins:wght@400;700&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+}
+
+.bigheader {
+  color: #fa3e23;
+}
+
+.salute {
+  color: #f74d1c;
+}
+
+@media only screen and (max-width: 768px) {
+  .social,
+  .desktop_seemore,
+  .logoLion,
+  .lelogo,
+  .chinisse {
+    display: none;
+  }
+
+  .middle_bar {
+    position: absolute;
+    height: 5px;
+    background-color: black;
+    width: 40%;
+    border: none;
+    top: 95vh;
+    margin-right: 5%;
+  }
+
+  .mobile_nav {
+    padding-bottom: 10%;
+  }
+
+  .open {
+    color: #525252;
+    font-size: 40px;
+    font-weight: bolder;
+    margin-left: 10px;
+  }
+
+  .close {
+    display: none;
+    color: white;
+    font-size: 20px;
+  }
+
+  .hamburger {
+    color: transparent;
+    padding: 0;
+    width: auto;
+    z-index: 100;
+    background: transparent;
+    cursor: pointer;
+    border: none;
+    margin-right: 10px;
+  }
+
+  .closingIcon {
+    position: fixed;
+    top: 1rem;
+    right: 1rem;
+    padding: 4px;
+    margin-top: 5%;
+  }
+
+  .hide_y_scroll {
+    overflow-y: hidden;
+  }
+
+  .link_item {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    color: white;
+  }
+
+  .menu_list {
+    box-sizing: border-box;
+    position: fixed;
+    display: none;
+    transform: translateY(-100%);
+    transition: transform 0.2s;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 99;
+    background: rgba(53, 55, 64, 0.863);
+    backdrop-filter: blur(5px);
+    color: white;
+    list-style: none;
+    padding-top: 4rem;
+    margin-top: 0;
+  }
+
+  li {
+    list-style: none;
+  }
+
+  .menu_list li {
+    margin-bottom: 5%;
+  }
+
+  .showMenu {
+    transform: translateY(0);
+    display: block;
+  }
+
+  .paragraph {
+    border: 4px solid #ececec;
+    padding: 20px 10px;
+    color: #7c7c7c;
+    background-color: white;
+  }
+
+  .message {
+    color: #7c7c7c;
+    text-align: center;
+  }
+
+  .place {
+    text-decoration: underline;
+  }
+
+  .banner {
+    padding-bottom: 12%;
+    text-align: center;
+  }
+
+  .partners {
+    background-color: #424146;
+    color: white;
```

Comment:
Kindly follow the [HTML&CSS Best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and try to DRY your css. Here white color repeating so, you can create a class with name white and give as a class attribute to HTML tags.

Reply:
- [ ] @Yacoubou-seidou as previously requested kindly create a class for your white color since its being repeated in various HTML elements.